src/model/spec/userProfile.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai';
+import * as deepFreeze from 'deep-freeze';
+import { UserProfile } from '../userProfile'
+import {} from 'mocha'
+import {} from 'core-js'
+
+describe('userProfile', () => {
+  it('Is instantiated and exists', () => {
+    // Arrange
+    // Act
+    const userProfile = new UserProfile();
```

Comment:
Instead of creating a new object, it is better to use enzyme features such using shallow, mount or render